lib/main.dart

```diff
@@ -1,115 +1,21 @@
 import 'package:flutter/material.dart';
 
+import 'screens/calc_screen.dart';
+
 void main() {
   runApp(const MyApp());
 }
 
 class MyApp extends StatelessWidget {
   const MyApp({super.key});
-
-  // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
```

Comment:
zacznij używać fat arrow-a (=>), żeby to zmienić kliknij na { Ctrl + . i naciśnij opcje 'Convert to expression body'
zamiast 
` ... {
return Widget
....
}`

będziesz miał 
`..... => Widget`

Reply:
linijka 3 i 12